app/src/main/java/edu/grinnell/grinnell_publications_android/Services/Interfaces/NetworkClientAPI.java

```diff
@@ -12,4 +19,91 @@
  * @see RemoteClientAPI
  */
 public interface NetworkClientAPI {
+
+    /* Publications */
+
+    /**
+     * @return  all the publications the current user is subscribed to
+     */
+    List<Publication> getSubscribedPublications();
```

Comment:
This should either take a user ID (probably not.....yet), or at least some identifier to indicate which publications the user is subscribed to.